cmd/skycoin/skycoin.go

```diff
@@ -486,6 +490,7 @@ func configureDaemon(c *Config) daemon.Config {
 	dc := daemon.NewConfig()
 	dc.Pex.DataDirectory = c.DataDirectory
 	dc.Pex.Disabled = c.DisablePEX
+	dc.Pex.Max = c.MaxIncomingConnections
```

Comment:
Is this the right setting to apply? Or do we need a new setting just for peer list size?

Reply:
The peerlist size controls the max incoming connections?

Reply:
I think need a new setting, the peer list doesn't control the incoming connections directly.